.github/workflows/flake8-pytest.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         # stop the build if there are Python syntax errors or undefined names
         # flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --max-complexity=10 --statistics
+        flake8 . --count --max-complexity=20 --statistics --max-line-length=99
```

Comment:
Use `black` instead

Reply:
Thanks for the suggestion @bart-of I will def. check it out! 🥳